copy.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+/*
+  copy text to clipboard
+  ref: https://github.com/mdn/webextensions-examples/tree/master/context-menu-copy-link-with-types
+
+*/
+
+var global_link_text = '';
```

Comment:
Do we really need a global variable? Purely factual inquiry, as I've not tried to override link copying before, so I don't know what the real way to do it is.

Reply:
this is another "kludge" that I brought from the original extension. Apparently this file and `background.js` are in different context, therefore cannot share data. Unless a global variable is used (awful) or another more elegant (that I still have to think of)